joint_trajectory_controller/include/joint_trajectory_controller/interpolation_methods.hpp

```diff
@@ -29,13 +29,15 @@ namespace interpolation_methods
 enum class InterpolationMethod
 {
   NONE,
-  VARIABLE_DEGREE_SPLINE
+  VARIABLE_DEGREE_SPLINE,
+  RUCKIG
```

Comment:
It might be confusing that splines run, followed by Ruckig. Maybe this should be renamed, SPLINES_THEN_RUCKIG?

```suggestion
  SPLINES_THEN_RUCKIG
```